CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## feature-multiOLVSettings - 2016-07-15
```

Comment:
Please move this desciption to `[Unreleased]` section into `Added` subsection (without specifying date of changes).